index.html

```diff
@@ -1,66 +1,354 @@
+                  </a>
+                </li>
+              </ul>
+            </li>
+        </ul>
+      </div>
+      <div class="grid-container" id="recent-works">
+          <div class="flex-container" id="title">
+            <h1>My Recent works</h1>
+          </div>
+          <div class="flex-container" id="multi-post">
+            <div class="placeholder" id="img-placeholder">
             </div>
-            <div id="normal-button">
-              <img id="menu-button" src="icons/Icon - Menu.svg" alt="menu-button"/>
+            <ul class="flex-container" id="multi-postBlock">
+              <li>
+                <h1 id="titlePost">Multi-Post Stories</h1>
+              </li>
+              <li>
+                <h1 id="mpSupportText">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</h1>
+              </li>
+              <li class="flex-container" id="mpTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="css">css</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+              </li>
+              <li>
+                <button class="button" type="button">
+                  <span id="buttonText">See Project</span>
+                </button>
+              </li>
+            </ul>
+          </div>
+          <div class="flex-container" id="cardWork">
+            <ul class="flex-container" id="cardElements">
+              <li>
+                <h1 id="cardTitle">Profesional Art Printing Data</h1>
+              </li>
+              <li>
+                <h1 id="cardDesc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h1>
+              </li>
+              <li class="flex-container" id="cardTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
             </div>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) we must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Kindly add more spacing to the top of the project cards  as defined in the Figma design

Your design
![image](https://user-images.githubusercontent.com/78251927/212377461-ab8e24f8-bcec-4542-80eb-3d2fdb7abb75.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/212377077-05a86f11-966c-4d8e-b99f-7d93c2c67f83.png)